.github/CODEOWNERS

```diff
@@ -160,4 +160,7 @@ Makefile                    @getsentry/owners-sentry-dev
 *sentry_app*.py                               @getsentry/ecosystem
 *sentryapp*.py                                @getsentry/ecosystem
 
+/api-docs/                                    @getsentry/ecosystem
```

Comment:
Interesting that one has a hyphen and the other doesn't 🤔

Reply:
I know, I think we briefly mentioned this and thought about changing it - if we do we'll need to update this as well.